docs: In-app-messaging - add limitation section (#5703)
invertase/react-native-firebase: docs/in-app-messaging/usage/index.md

This module provides a JavaScript API to allow greater control of the displaying of these messages.

# Limitations

> Accorrding to [Gihthub isssue](https://github.com/firebase/firebase-ios-sdk/issues/4768) Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch - this is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (ie: Analytics event or problematically triggered FIAM campaigns), there is no daily rate limit.

Reviewer: Do you mean `programmatically` triggered vs `problematically` ? I ask because with software you never know, there could be another issue causing campaign triggering to be problematic as well 😅

> Accorrding to [Gihthub isssue](https://github.com/firebase/firebase-ios-sdk/issues/4768) Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch - this is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (ie: Analytics event or problematically triggered FIAM campaigns), there is no daily rate limit.

Reviewer: ```suggestion
Accorrding to [Gihthub isssue](https://github.com/firebase/firebase-ios-sdk/issues/4768) Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch. This limit is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (for example: Analytics event or programmatically triggered in-app-messaging campaigns), there is no daily rate limit.
```

> According to [Github isssue](https://github.com/firebase/firebase-ios-sdk/issues/4768) Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch. This limit is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (for example: Analytics event or programmatically triggered in-app-messaging campaigns), there is no daily rate limit.

Reviewer: ```suggestion
According to github issue https://github.com/firebase/firebase-ios-sdk/issues/4768 Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch. This limit is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (for example: Analytics event or programmatically triggered in-app-messaging campaigns), there is no daily rate limit.
```

> 
> 

Reviewer: ```suggestion

```

This module provides a JavaScript API to allow greater control of the displaying of these messages.

> # Limitations
> According to [Github isssue](https://github.com/firebase/firebase-ios-sdk/issues/4768) Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch. This limit is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (for example: Analytics event or programmatically triggered in-app-messaging campaigns), there is no daily rate limit.

Reviewer: ```suggestion
# Limitations

According to [Github isssue](https://github.com/firebase/firebase-ios-sdk/issues/4768) Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch. This limit is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (for example: Analytics event or programmatically triggered in-app-messaging campaigns), there is no daily rate limit.

```

> Accorrding to [Gihthub isssue](https://github.com/firebase/firebase-ios-sdk/issues/4768) Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch. This limit is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (for example: Analytics event or programmatically triggered in-app-messaging campaigns), there is no daily rate limit.

Reviewer: ```suggestion
According to [Github isssue](https://github.com/firebase/firebase-ios-sdk/issues/4768) Firebase In-App Messaging allows only 1 campaign per day on app foreground or app launch. This limit is to prevent you from accidentally overwhelming your users with non-contextually appropriate messages. However, if you use the contextual triggers (for example: Analytics event or programmatically triggered in-app-messaging campaigns), there is no daily rate limit.
```
